docker-hub/repos.md

```diff
@@ -128,6 +128,23 @@ To view individual tags, click on the **Tags** tab.
 
 ![Manage Repo Tags](images/repo-tags-list.png)
 
+An image is considered stale if there has been no push/pull activity for more
+than 1 month, i.e.:
+
+* It has not been pulled for more than 1 month
+* And it has not been pushed for more than 1 month
+
+A multi-architecture image is considered stale if all single-architecture images
+part of its manifest are stale.
+
+To delete a tag, select the corresponding checkbox and select the action **Delete**
```

Comment:
```suggestion
To delete a tag, select the corresponding checkbox and select **Delete** from the **Action** drop-down list.
```

Reply:
Ah, thx, yes, that reads better. Updated 👍